integreat_cms/cms/templates/_form_language_tabs.html

```diff
@@ -96,6 +96,11 @@
                                         <span title="{% translate "Translation missing" %}" class="no-trans">
                                             <i icon-name="x"></i>
                                         </span>
+                                    {% elif other_status == translation_status.MACHINE_TRANSLATED %}
+                                        <span title="{% translate "Machine translated" %}"
+                                              class="no-trans">
```

Comment:
```suggestion
                                        <span title="{% translate "Machine translated" %}">
```
Ah, so sorry, and I completely missed the incorrect class :innocent:

Reply:
Thank you!